views/pages/results.html

```diff
@@ -33,7 +33,7 @@
     {{#records}}
     <li>
       <article>
-        <a href="/details/{{system-number}}">
+        <a href="/details/{{system-number}}?{{querystring}}&multipleResults=true">
```

Comment:
URL escape issue again.

I wonder if multipleResults should just be a flag ideally. Just thinking that setting it to "false" will behave the same as if it is set to "true" or anything else.

This should do for now though of course if you like.